libraries/AP_Logger/AP_Logger_Backend.cpp

```diff
@@ -573,6 +573,11 @@ bool AP_Logger_Backend::Write_VER()
         patch: fwver.patch,
         fw_type: fwver.fw_type,
         git_hash: fwver.fw_hash,
+#ifdef APM_BUILD_DIRECTORY
+        build_type: APM_BUILD_DIRECTORY,
+#else
+        build_type: 0,
+#endif
```

Comment:
```suggestion
        build_type: fwver.vehicle_type,
```
This is in `AP_FWVersion` as `vehicle_type`

Reply:
thanks! fixed